src/chrome/playSound.js

```diff
@@ -0,0 +1,20 @@
+import { SOUND_URL, SOUND_REPEAT } from '../constants';
+
+// Get sound options and play if enabled
+
+export const getAndPlaySoundNotificationOption = (store, mainAudioPlayer) => {
```

Comment:
btw window.Audio doesn't get along with sinon.spy so I decided to go w/ MockAudio class because it works great.   Also minor adjustment to pass the lint and test.